.binder/environment.yml

```diff
@@ -3,28 +3,25 @@ channels:
   - defaults
   - conda-forge
 dependencies:
-  - numpy==1.18.4
-  - mkl=2020.0=166
-  - mkl_fft=1.0.15=py36ha843d7b_0
-  - pip==20.1
-  - python=3.6.10=hcf32534_1
-  - scipy=1.4.1=py36h0b6359f_0
-  - six=1.14.0=py36_0
+  - python=3.9.5
+  - numpy==1.21.0
+  - scipy==1.7.0
+  - pip=21.1.2
+  - six==1.16.0
+  - holoviews==1.13.2
```

Comment:
Maybe current issues with holoviews on Binder come from it being conda installed instead of pip installed somehow?